skimage/data/__init__.py

```diff
@@ -592,6 +592,63 @@ def checkerboard():
     return _load("data/chessboard_GRAY.png")
 
 
+def cells3D():
+    """3D microscopy images of cells.
+
+    The data for this was provided by the Allen Institute for Cell Science.  It
+    has been downsampled by a factor of 4 in the row and column dimensions to
+    reduce computational time.
+
+    The microscope reports the following voxel spacing
+
+        * Original voxel size is ``(0.290, 0.065, 0.065)``.
+        * Scaling factor is ``(1, 4, 4)`` in each dimension.
+        * After rescaling the voxel size is ``(0.29 0.26 0.26)``.
+
+    The returned data is a 3D array with the physical dimensions provided in
+    ``(z, y, x)`` order.
+
+    Returns
+    -------
+    cells3D: (60, 256, 256) float64 ndarray
+        The volumetric images of cells taken with an optical microscope.
+    """
+
+    return _load("data/cells.tif")
+
+
+def human_mitosis():
+    """A dataset containing an image of human cells undergoing mitosis.
+
```

Comment:
1 space only, maybe?

Reply:
@alexdesiqueira personally I don't understand this suggestion ^^

Reply:
I think it is outdated.

Reply:
@mkcor I meant two blank lines -> one blank one :slightly_smiling_face: Wasn't clear enough, sorry about that.